src/content/release-notes.mdx

```diff
@@ -5,6 +5,35 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.21
+
+7 June 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### New Features
+
+- The platform will now only obfuscate variables that are longer than 5 characters  <!-- 8090 -->
+- We've added the ability to [set custom NodeSelectors](self-hosted/helm-configuration.mdx#globals)  <!-- 8093 -->
+
+### Improvements
+
+- Show GitHub installations error notifications only when opening GitHub tab on Deploy dialog  <!-- 8106 -->
+- Handle suspended GitHub installation error on the branch selector in the Okteto UI  <!-- 8123 -->
+- Enable more error variant feedback on the GitHub BranchSelector in the Okteto UI  <!-- 8122 -->
+- Handle suspended Github app installation on the repositories selector in the Okteto UI  <!-- 8128 -->
+- Removed ingress name validation in the mutation webhook for ingress created as part of cert-manager challenges
+- Handle HTTP 5xx errors more gracefully in the Okteto UI  <!-- 8149 -->
+- Okteto CLI: Changed .oktetodeployignore files to use .oktetoignore for [remote executions of deploy and destroy](core/remote-execution.mdx#ignoring-files), which allows more granularity to ignore files. You can add global ignore, but also specific ignores per operation
```

Comment:
Should we include a bullet point for the CLI version used in the backend? We've been setting it in previous release notes but I don't know if it provides value for customers